CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- EGLBufferReader now checks if buffers are alive beffore using them.
```

Comment:
There is a typo at "beffore", also could you make a `### Bugfixes` subsection including your change?

Reply:
fixed, sorry about that.